extension/src/Runner/testRunnerEnvironment.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { TestSuite } from "../protocols";
+
+export interface ITestRunnerEnvironment {
+    tests: TestSuite[];
+    isDebugMode: boolean;
+    storagePath: string;
+    port: number | undefined;
+    transactionId: string | undefined; // TODO: remove later after refactoring logger
+}
+
+export class JarFileTestRunnerEnvironment implements ITestRunnerEnvironment {

```

Comment:
This class does not have any member functions. If it's just for type checking, you can use another extending interface which has the extra 'classpathStr' member.

Reply:
nice